lib/astarte_vmq_plugin/config.ex

```diff
@@ -67,6 +68,38 @@ defmodule Astarte.VMQ.Plugin.Config do
     Application.put_env(:astarte_rpc, :amqp_connection, astarte_rpc_amqp_opts)
   end
 
+  defp init_ssl_options(amqp_options) do
+    ssl_options = Application.get_env(:astarte_vmq_plugin, :amqp_ssl, [])
+    build_ssl_options(ssl_options, amqp_options)
+  end
+
+  defp build_ssl_options([], amqp_options), do: amqp_options
+
+  defp build_ssl_options(ssl_options, amqp_options) do
+    case Keyword.fetch(ssl_options, :disable_sni) do
+      {:ok, false} ->
+        populate_ssl_options(ssl_options, amqp_options)
+
+      _ ->
+        amqp_options
```

Comment:
This would disable SSL instead of disabling SNI I think, they're two different things